### ISSUE TITLE
real-test.py

### ISSUE THREAD

**Issue by Johnweii25** [caiyuanhao1998/MST#53]
when running the test.py,some situations occured in 'out=model(input,mask_3d_shift,mask_3d_shift_s)',the model that used in test was trainned by me before testting.
![微信图片_20240118154502](https://github.com/caiyuanhao1998/MST/assets/147235455/07d05e0b-5b59-4575-8830-05e9b9b4bd4f)
and the system has been changed into ubuntu,if you could give me some advice?thanks

**Comment by Johnweii25** [comment]
sorry to bother you again,when im trainning in mst_m,'userwarning:the epoch parameter in "scheduler.step()" was not necessary...',is there something wrong?is the trained models count on?
![Screenshot from 2024-01-19 01-07-36](https://github.com/caiyuanhao1998/MST/assets/147235455/275b1f27-1437-4c03-93d1-c9dbb3d902db)

**Comment by caiyuanhao1998** [comment]
(i) You give too much inputs to the trained model.

(ii) No worry

**Comment by Johnweii25** [comment]
> (i) You give too much inputs to the trained model.
> 
> (ii) No worry

you are so kind!and appreciate your works!!!
